packages/experimental/MenuButton/src/MenuButton.tsx

```diff
@@ -84,10 +35,10 @@ export const MenuButton = compose<MenuButtonType>({
 
       return (
         <Slots.root>
-          <Button {...buttonProps}>
+          <Slots.button {...mergedProps} primary={primary}>
```

Comment:
I don't think primary has to be a separate prop here.

Reply:
I destructured primary before, to use it for styling of the icon. That's why I need to add it here separately